Xamarin.Forms.Core/ListView.cs

```diff
@@ -46,6 +46,8 @@ public class ListView : ItemsView<Cell>, IListViewController, IElementConfigurat
 
 		public static readonly BindableProperty SeparatorColorProperty = BindableProperty.Create("SeparatorColor", typeof(Color), typeof(ListView), Color.Default);
 
+		public static readonly BindableProperty SpinnerColorProperty = BindableProperty.Create(nameof(SpinnerColor), typeof(Color), typeof(ListView), Color.Black);
```

Comment:
I think you should use the `Color.Default`and in the renderers you need to capture the default color of the spinner when they are initialised , so if the user changes back to the default color you can set the default color of the platform.

Reply:
I did `Color.Default`, but then the spinner disappeared for some weird reason, at least on iOS. I'll check it again.